### ISSUE TITLE
cli feature idea: list generator tasks / help and list available generators

### ISSUE THREAD

**Issue by tunnckoCore** [generate/generate#18]
Would be good to be able to list generator tasks - not all, but maybe that are not silenced? only the main, without the aliases. Also generator help task if exist.

Something like that

```sh
# list generate-project tasks
$ gen project --tasks

# list generate-node tasks
$ gen node --tasks
$ gen node --list-tasks?
$ gen node --list?

# list 
$ gen foo bar --tasks
$ gen foo bar baz qux --tasks
```

Hm. So, maybe would make more sense if I rephrase it to per generator options.

And just popup one more thing in my mind - list available generators (globally/locally installed).

Maybe something like

```sh
# list globally installed generators
$ gen --list

# list locally installed
$ gen --list-local

# list sub-generators of `foo` generator
$ gen foo --list

# list sub-sub-generator
$ gen foo bar baz --list
```

**Comment by jonschlinkert** [comment]
I just realized the [docs on this repo](https://github.com/node-base/task-prompts/blob/master/index.js#L58) suck, but it has a method that prompts for tasks
